blogposts/2021/how-to-not-break-a-search-engine-or-what-i-learned-about-unglamorous-engineering.md

```diff
@@ -0,0 +1,297 @@
+---
+title: "How not to break a search engine or: What I learned about unglamorous engineering"
+description: "When we switched to a new search query parser in September 2020, you'd never know that
+anything had changed. This is an account of the invisible, rigorous testing that
+happened behind the scenes to ensure a seamless transition."
```

Comment:
```suggestion
description: "When we switched to a new search query parser in September 2020, you'd never know that
anything had changed. This is an account of the unglamorous engineering that
happened behind the scenes to ensure a seamless transition."
```

Since "behind the scenes" implies "invisible" I played with this idea but don't think it's as nice. I think it's good that the short description has a topic of 'testing', and the rest of the article will deal with the unglamorous stuff well enough so, yeah, this was an idea but I'm happy as it is.

Reply:
@rvantonder You are totally right about invisible + behind the scenes being tautological though, I'm going to tweak it!